src/Lecture_3/Homework_3/Task_10_Not_Completed.java

```diff
@@ -0,0 +1,22 @@
+package Lecture_3.Homework_3;
+
+public class Task_10_Not_Completed {
+
+    public static void main(String[] args) {
+
+        int [] array = {4, 16, 7, 8, 12, 23, 56, 64};
+        int start = 0;
+        int end = array.length -1;
+        int length = array.length;
+        boolean isZigZag = false;
+
+        if (array.length % 2 == 0) {
+            for (int i = start; i < length; i++) {
+                if (i % 3 == 0) {
+                    // check that first/previous element is smaller than next, and then check that second is bigger than
```

Comment:
You have the solution as part of the cod efor the next lecture

Reply:
Got it, just wanted to try to resolve it myself.